### DISCUSSION TITLE
errortext....http://teslalogger:5010/getallcars

### DISCUSSION THREAD

**Discussion by msman0** [bassmaster187/TeslaLogger#1329] [Q&A]
Hallo, bei mir funktioniert es plötzlich nicht mehr. Es erscheint folgender Fehler:
errortext = 'Error: php_network_getaddresses - URL: http://teslalogger:5010/getallcars'

Eine Änderung am System habe ich nicht gemacht (Diskstation). Auch ein Reboot hat leider nicht geholfen.

Wie kann der Fehler behoben werden?
Danke!

**Comment by andrelx** [comment]
Simply Update the Teslalogger to make sure its at the latest version, then go into settings and reset your Mytesla credentials.

**Reply by msman0** [reply]
I have version 1.58.7.0 and it display that version 1.59.0.0 is available. If I click update it says to reboot but the version is not updated. Go to settings-My tesla only the same error message as above displayed. Nothing to enter credentials etc.
What can I do else? 
Thank you very much!

**Comment by msman0** [comment]
Is there no chance to fix the error? How can I force the update?
Thank you.

**Comment by msman0** [comment]
I have done the update manual and now it work again!
